Mergin/plugin.py

```diff
@@ -199,6 +203,7 @@ def on_config_changed(self):
         """Called when plugin config (connection settings) were changed."""
         self.create_manager()
         self.enable_toolbar_actions()
+        self.post_login()
```

Comment:
How does it work in plugin with auto-refresh of auth bearer token (e.g. QGIS session is open for several days)?
If we have it, I would consider checking attention_required as well.

Reply:
Probably I would not tie auto-refresh of token to the action required check - I would suggest to keep it simple and not worry too much about this case of qgis being open for several days...